c_api/Makefile

```diff
@@ -104,11 +110,92 @@ release-linux-gnu-x86:
 
 release-linux-gnu-arm:
 	rustup target add aarch64-unknown-linux-gnu
-	cargo build --target=aarch64-unknown-linux-gnu  $(CARGO_BUILD_FLAGS)
+	sudo apt-get update && sudo apt-get install -y gcc-aarch64-linux-gnu libc6-dev-arm64-cross
+	CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER=aarch64-linux-gnu-gcc cargo build --target=aarch64-unknown-linux-gnu  $(CARGO_BUILD_FLAGS)
 	@$(call make_zip,$(TARGET_DIR)/aarch64-unknown-linux-gnu/release,aarch64-unknown-linux-gnu)
 
+release-ios-arm64:
```

Comment:
Could you add a comment for what each target compiles? It looks like some are specific to the simulator?

Reply:
I added comments for each of the mobile targets in the Makefile